app/src/main/java/org/fossasia/openevent/general/search/SearchFilterFragment.kt

```diff
@@ -0,0 +1,86 @@
+package org.fossasia.openevent.general.search
+
+import android.graphics.drawable.Drawable
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.Menu
+import android.view.MenuInflater
+import android.view.MenuItem
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import androidx.navigation.Navigation
+import androidx.navigation.fragment.navArgs
+import kotlinx.android.synthetic.main.fragment_search_filter.view.freeStuffCheckBox
+import org.fossasia.openevent.general.R
+import org.fossasia.openevent.general.utils.Utils
```

Comment:
Import just required functions not entire class

Reply:
resolved Thanks :)